File: programovani/ulohy/files/modelovani-obrazcu-ii/trojuhelnik-180-pomocna.js
// Deklarace a inicializace
const vel = 6; // Inicializace velikosti trojuhelniku
pom = 0; // Inicializace pomocne promenne

// Vnejsi cyklus pro radky
for(var i = 1; i <= vel; i++) {

    // Inicalizujeme promennou pro vypsani radky
    var rad = "";

    // Vnitrni cyklus pro jednotlive hvezdicky
    for(var j = 1; j <= vel; j++) {

        // V pripade, ze budeme v limitu
        if(j <= pom)

            // Vypiseme mezery
            rad += "  ";

        // V opacnem pripade
        else

            // Vypiseme hvezdicky
            rad += "* ";

    }            

    // Inkrementujeme pomocnou promennou
    pom++;

    // Vypiseme radku
    console.log(rad);

}

File: programovani/ulohy/files/modelovani-obrazcu-ii/trojuhelnik-180-pomocna.py
# Deklarace a inicializace
vel = 6 # Inicializace velikosti trojuhelniku
pom = 0 # Inicializace pomocne promenne

# Vnejsi cyklus pro radky
for i in range(1, vel + 1):
    
    # Vnitrni cyklus pro jednotlive hvezdicky
    for j in range(1, vel + 1):

        # V pripade, ze budeme v limitu
        if(pom >= j):

            # Vypiseme mezeru
            print("  ", sep="", end="")

        # V opacnem pripade
        else:

            # Vypiseme hvezdicky
            print("* ", sep="", end="")

    # Inkrementujeme pomocnou promennou
    pom += 1

    # Vypiseme novou radku
    print(end="\n")
